com.theappgineer.community_remote.yml

```diff
@@ -0,0 +1,29 @@
+id: com.theappgineer.community_remote
```

Comment:
`theappgineer.com` seems to be a parked domain. Is it under the control of the app developers? How can we check this?

Reply:
I indeed parked the domain. I can share relevant parts of the latest invoice, if there is a private channel available for this.

Reply:
putting proper content would be enough.

Reply:
Is it really necessary to host something? Like I said, I parked it but there is no hosting attached to it. Sharing a document that connects me to the domain should be enough, right?

Reply:
Yes, having a simple webpage with a link would be appreciated

Reply:
I can try setting up a redirect to https://github.com/theappgineer. Would that close the loop?

Reply:
Yes that'd work.

Reply:
Redirect is in place, setup https://theappgineer.github.io/ to redirect to.

Reply:
Thanks